src/components/hv-view/index.js

```diff
@@ -41,142 +47,213 @@ export default class HvView extends PureComponent<HvComponentProps> {
 
   props: HvComponentProps;
 
-  render() {
-    let viewOptions = this.props.options;
-    const { skipHref } = viewOptions || {};
-    const props: InternalProps = createProps(
+  attributes: Attributes;
+
+  constructor(props: HvComponentProps) {
+    super(props);
+    this.updateAttributes();
+  }
+
+  componentDidUpdate(prevProps: HvComponentProps) {
+    if (prevProps.element === this.props.element) {
+      return;
+    }
+
+    this.updateAttributes();
+  }
+
+  updateAttributes = () => {
+    // $FlowFixMe: reduce returns a mixed type, not Attributes
```

Comment:
It seems to only retrieve fields defined in ATTRIBUTES, so why is the return type not Attributes? s it throwing a type error?

Reply:
Yes, it's throwing a typing error. The problem appear to be with the `reduce` operation, which contains an inner function and might be too complicated to infer from Flow. There might be ways to solve that kind of typing error but I haven't found one yet.